src/Compatibility.StorageTable.AcceptanceTests/SetupFixture.cs

```diff
@@ -11,49 +12,37 @@
     public class SetupFixture
     {
         [OneTimeSetUp]
-        public async Task OneTimeSetUp()
+        public Task OneTimeSetUp()
         {
             var connectionString = this.GetEnvConfiguredConnectionStringByCallerConvention();
 
             var account = CloudStorageAccount.Parse(connectionString);
             TableClient = account.CreateCloudTableClient();
 
-            allSagaDataTypeNames = GetType().Assembly.GetTypes().Where(x => x.GetInterfaces().Contains(typeof(IContainSagaData)))
-                .Select(x => x.Name).ToArray();
+            TablePrefix = $"{Path.GetFileNameWithoutExtension(Path.GetTempFileName())}{DateTime.UtcNow.Ticks}".ToLowerInvariant();
 
-            foreach (var dataTypeName in allSagaDataTypeNames)
+            allConventionalSagaTableNamesWithPrefix = GetType().Assembly.GetTypes().Where(x => x.GetInterfaces().Contains(typeof(IContainSagaData)))
+                .Select(x => $"{TablePrefix}{x.Name}").ToArray();
+
+            return Task.WhenAll(allConventionalSagaTableNamesWithPrefix.Select(async tableName =>
```

Comment:
Similar to the question about one time tear down

Reply:
Happy to align